os/arch/arm/src/s5j/Kconfig

```diff
@@ -56,6 +56,7 @@ config S5J_HAVE_ADC
 
 config S5J_HAVE_I2C
 	bool
+	select ARCH_HAVE_I2C if S5J_I2C
```

Comment:
This means this chip has I2C hw. Regardless of enabling I2C by user, it should select ARCH_HAVE_I2C.